tests/test_utils.py

```diff
@@ -55,6 +56,19 @@ def test_write_labels(tmpdir):
     assert(label == utils.read_labels(file_name=file_path))
 
 
+def test_get_ImageDataGenerator():
+    img_name = next(f for f in os.listdir('data/') if f.endswith('.png'))
+    img = utils.load_image(img_name)
```

Comment:
Let's create an image in memory here, rather than loading a real image from the data directory (see the `test_load_image` test for an example).